Runtime/Lobbies/LobbyMember.cs

```diff
@@ -16,7 +16,7 @@ public class LobbyMember : INetworkConnection, IMetadataContainer, IDisposable {
   public static IMessageProcessor MessageProcessor;
 
   static LobbyMember() {
-    MessageProcessor = new LZFCompressor();
+    MessageProcessor = null;
```

Comment:
Any particular reason this was removed?

Reply:
Using LZFCompressor instantly crashes Unity. I'm not sure where it goes wrong so I just commented it out.

Reply:
OK, that's a sign we have a memory access violation somewhere in our code. I think we can keep this for now, but please leave a TODO(james7132) comment to investigate this. The LZF compression is part of the reason why we have been able to keep our bandwidth usage low despite sending a network message basically every tick in Backroll.